neurom/stats.py

```diff
@@ -34,11 +34,28 @@
 from collections import OrderedDict
 from scipy import stats as _st
 import numpy as np
-
+from enum import Enum, unique
 
 FitResults = namedtuple('FitResults', ['params', 'errs', 'type'])
 
 
+@unique
+class Stat_tests(Enum):
```

Comment:
From [pep8 naming conventions](https://www.python.org/dev/peps/pep-0008/#class-names), maybe better to use CamelCase here.